src/confidential.rs

```diff
@@ -647,6 +720,68 @@ impl<'de> Deserialize<'de> for Nonce {
     }
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+/// Blinding factor used for asset commitments.
```

Comment:
These comments are usually on top of the derives.

Reply:
Ooops.